src/DB/exerciseData.json

```diff
@@ -316,9 +316,8 @@
 {
   "exercise": "Romanian Deadlift",
   "instructions": [
-    "Stand with feet shoulder-width apart, hinge at the hips while keeping a straight back,"
-    "lower the weight close to the body, and drive through the heels to return. Maintain a "
-    "neutral spine. Strengthen hamstrings, glutes, and lower back with this effective exercise."
+    "Stand with feet shoulder-width apart, hinge at the hips, while keeping a straight back,"
```

Comment:
You're missing a comma right here, should be `"Stand with feet shoulder-width apart, hinge at the hips, while keeping a straight back,",`